src/views/MyPR.vue

```diff
@@ -21,6 +21,9 @@
                     <div class="heading4 text-white">
                         <span>{{ user.time }}</span>
                     </div>
+                    <div>        
+                        <img @click="delete_pr(user.id), update_score(user.uid, user.difficulty)" :id="set_id(user.id)" src="../assets/delete-icon.svg" style="&:hover{background-color: aqua; cursor: pointer;}; height: 1.5rem; width: 1.5rem;" alt="delete">
+                    </div>
```

Comment:
Also when the users presses on the delete button, show him a modal that once this is deleted, the action cant be undone, something like that and have options like continue or cancel because someone might accidentaly press the delete button.

So before deleting add a confirmation modal.